hooked/views.py

```diff
@@ -72,9 +73,12 @@ def check_webhook_signature(self, request):
             return
 
         self.log_failure(
-            'Mismatch between computed and claimed signature of incoming '
-            'events. I computed {}, but the HTTP header said I should '
-            'expect to find {}'.format(computed_signature, claimed_signature))
+            message='Mismatch between computed and claimed signature of incoming events',
+            extra={
+                'expected_signature': computed_signature,
+                'actual_signature': claimed_signature
+            }
+        )
```

Comment:
Ahh, yes - we would miss those.  Come to think of it though, are those values useful enough to warrant logging?  I mean we know that they're hashes and they they don't match - is there anything we'd want to do with those values that would necessitate logging them?  Maybe it's just practice, I don't actually know so I will defer to someone with a greater knowledge than me on this one.